flake.nix

```diff
@@ -1,18 +1,23 @@
 {
   description = "ocaml-packages-overlay";
 
-  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
does the bot know to change this to `unstable-small`?

Reply:
if I read the code in the auto-update.js file correctly, we're losing the ability to update to an unstable-small channel.

Reply:
[ffd99fa](https://github.com/anmonteiro/nix-overlays/pull/253/commits/ffd99fa9bb5ba3d56ebc80a1ce0f0796c0a26c52) proposes using `?rev` which is way more explicit to know which pkgs hash we're on.

Reply:
I'll leave it to you to update `auto-update.js` accordingly.

Reply:
Done